bookkeeper-server/src/main/java/org/apache/bookkeeper/bookie/Bookie.java

```diff
@@ -1560,6 +1560,15 @@ public static boolean format(ServerConfiguration conf,
                     }
                 }
             }
+
+            // Clean up metadata directories if they are separate from the
+            // ledger dirs
+            File metadataDir = new File(conf.getGcEntryLogMetadataCachePath());
```

Comment:
is this code to be executed only if the feature is enabled?

Reply:
yes, it uses rocksDB to cache metadata map only if the feature is enabled else it continues to use existing logic.